src/Makefile

```diff
@@ -0,0 +1,7 @@
+.PHONY: help
+help:  ## You're looking at it
+	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-10s\033[0m %s\n", $$1, $$2}'
+
+.PHONY: test
+test:  ## Run unit tests
+	tox -e xenial
```

Comment:
i don't see a xenial target in tox.ini

Reply:
This Makefile is for the `/src` directory and refers to the existing [`[testenv:xenial]`](https://github.com/Yelp/nerve-tools/blob/14385148f27b9367aab8806809e543a079bbb731/src/tox.ini#L16)